internal/integration/docker-compose.yaml

```diff
@@ -120,4 +120,13 @@ services:
     healthcheck:
       test: mysqladmin ping -ppass
     ports:
-      - 4308:3306
\ No newline at end of file
+      - 4308:3306
+
+# Default DB test, No Password
+  tidb:
+    platform: linux/amd64
+    image: pingcap/tidb:v5.4.0
+    healthcheck:
+      test: mysqladmin ping -ppass
```

Comment:
are you sure the image contains `mysqladmin`?

Reply:
there isn't - removed